applications/MedApplication/custom_io/med_model_part_io.cpp

```diff
@@ -498,20 +638,51 @@ void MedModelPartIO::ReadModelPart(ModelPart& rThisModelPart)
             rThisModelPart.CreateNewGeometry(kratos_geo_name,
                                              ++num_geometries_total,
                                              geom_node_ids);
+
+            if (groups_by_fam.empty()) {continue;} // file does not contain families
+            const int fam_num = geom_family_numbers[i];
+            if (fam_num == 0) {continue;} // geometry does not belong to a SubModelPart
```

Comment:
how can an object not belong to a family? Is it in the root family in this case? Or is it not part of the mesh?

Reply:
they all belong to a family. Family zero is where everything is that is not part of a group (aka SubModelPart).

The check in line 642 is if no family information was written to the file at all, the one in 644 if the entity only belongs to the main mesh but not any submodelparts

does that answer the question?

Reply:
yup, got it. Thanks!